scripts/ci-e2e-lib.sh

```diff
@@ -79,7 +79,7 @@ k8s::resolveVersion() {
   fi
 
   if [[ "$version" =~ ^ci/ ]]; then
-    resolveVersion=$(curl -LsS "http://gcsweb.k8s.io/gcs/kubernetes-release-dev/ci/${version#ci/}.txt")
+    resolveVersion=$(curl -LsS "http://gcsweb.k8s.io/gcs/k8s-release-dev/ci/${version#ci/}.txt")
```

Comment:
Is this intentional?

Reply:
Yes, based on the suggestion from @fabriziopandini in https://github.com/kubernetes-sigs/cluster-api/issues/4103#issuecomment-764918808

Reply:
As per @neolit123 comment https://github.com/kubernetes-sigs/cluster-api/issues/4103#issuecomment-766402707 we should punt on this change 😢

Reply:
Sure, done.